darwin-menu.js

```diff
@@ -0,0 +1,135 @@
+module.exports = function menu (app, mainWindow) {
```

Comment:
Looking through the functions here, we can see that we need both the `app` and `mainWindow` from the `main.js` file.